rpcs3/rpcs3qt/emu_settings.cpp

```diff
@@ -265,6 +265,7 @@ bool emu_settings::ValidateSettings(bool cleanup)
 void emu_settings::SaveSettings() const
 {
 	YAML::Emitter out;
+	m_current_settings.remove("VFS");
```

Comment:
@Megamouse Siffices?

Reply:
No. Try the thing I did when I removed the settings that aren't existing anymore after asking for user confirmation

Reply:
It should only be done when if (game)